### ISSUE TITLE
test failures on 10.3.3 with --cov on Windows

### ISSUE THREAD

**Issue by mykmelez** [tapjs/tapjs#373]
Upgrading [mozilla/qbrt](https://github.com/mozilla/qbrt)'s tap dependency version from 10.3.2 to 10.3.3 causes its tests to fail with "Error: Cannot find module" when run with `--cov` on Windows:

```
$ node_modules/.bin/tap test/package-path.js --cov
module.js:471
    throw err;
    ^

Error: Cannot find module 'C:\Users\myk\.node-spawn-wrap-4664-85d6b0f6791b\node.EXE'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:390:7)
    at startup (bootstrap_node.js:150:9)
    at bootstrap_node.js:505:3
test/package-path.js .................................. 0/1
  not ok test/package-path.js
    timeout: 240000
    file: test/package-path.js
    command: 'c:\Program Files\nodejs\node.EXE'
    args:
      - test/package-path.js
    cwd: 'c:\Users\myk\Projects\qbrt'
    exitCode: 1

total ................................................. 0/1


  0 passing (133.489ms)
  1 failing

----------|----------|----------|----------|----------|----------------|
File      |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
----------|----------|----------|----------|----------|----------------|
All files |  Unknown |  Unknown |  Unknown |  Unknown |                |
----------|----------|----------|----------|----------|----------------|
```

Running without `--cov` is unaffected, as is running with `--cov` on Linux/Mac. The problem only occurs when `--cov` is specified on Windows.

**Comment by morrisallison** [comment]
~~This also occurs on Ubuntu and OS X. The failures start [when `nyc` was bumped to v11](https://github.com/tapjs/node-tap/commit/12255f8092190dd600dcf3335000ec14aa80ff60). No failures with `nyc` v10.~~

**Comment by mykmelez** [comment]
@morrisallison Hmm, I don't see the same issue on Linux/macOS. Can you paste/link to the error message(s) you're getting on those platforms?

FWIW, I traced the Windows-specific failure to https://github.com/tapjs/spawn-wrap/issues/56, and I've requested a pull for a fix in https://github.com/tapjs/spawn-wrap/pull/57.

**Comment by morrisallison** [comment]
@mykmelez My mistake, the new `nyc` version broke [`babel-tap`](https://github.com/jkrems/babel-tap) with very similar errors and steps to reproduce. Using `tap` with `babel`, without `babel-tap` works as expected.

**Comment by mykmelez** [comment]
This was fixed downstream in https://github.com/istanbuljs/nyc/commit/8768afe567371f16974a58111c9ff10c222f8645. I've requested updating to the fixed version of nyc in #394.

**Comment by al-k21** [comment]
Hello, I guess this need to be reopened.

- Semver module is  using [tap@v10.7.0](https://github.com/npm/node-semver/blob/v5.5.0/package.json#L10), which seems to still be broken:
```
 Error: Cannot find module 'C:\Users\Administrator\.node-spawn-wrap-2268-b8f7c9a4ab2d\node.EXE'
     at Function.Module._resolveFilename (module.js:538:15)
     at Function.Module._load (module.js:468:25)
     at Function.Module.runMain (module.js:684:10)
     at startup (bootstrap_node.js:187:16)
     at bootstrap_node.js:608:3
 module.js:540
     throw err;
```

- Sax module is using [tap@v10.5.1](https://github.com/isaacs/sax-js/blob/v1.2.4/package.json#L23), which is also broken:
```
'C:\Users\Administrator\.node-spawn-wrap-2036-57e6e875908f\node.EXE'
     at Function.Module._resolveFilename (module.js:538:15)
     at Function.Module._load (module.js:468:25)
     at Function.Module.runMain (module.js:684:10)
     at startup (bootstrap_node.js:187:16)
     at bootstrap_node.js:608:3
 module.js:540
     throw err;
```

/cc @mykmelez @isaacs
